src/cljc/matcher_combinators/ansi_color.cljc

```diff
@@ -7,19 +7,20 @@
   *use-color*
```

Comment:
would be nice to move this also to `matcher-combinators.config` but would be breaking in case anybody uses it with `binding`